cardano-cli/cardano-cli.cabal

```diff
@@ -32,7 +32,9 @@ common project-config
                         -Wno-unticked-promoted-constructors
                         -Wpartial-fields
                         -Wredundant-constraints
-                        -Wunused-packages
+
+  if impl(ghc >= 9.6)
+    ghc-options:        -Wunused-packages
```

Comment:
Why? Tbh it's quite useful feature

Reply:
No idea.  One of the packages gets flagged on `ghc-8.10.7` but when I remove it, the compile fails.  We still have it for `ghc-9.6` so CI will catch it for us.